eventsocket/eventsocket.go

```diff
@@ -154,31 +155,39 @@ func (h *Connection) readLoop() {
 // readOne reads a single event and send over the appropriate channel.
 // It separates incoming events from api and command responses.
 func (h *Connection) readOne() bool {
-	hdr, err := h.textreader.ReadMIMEHeader()
+	var (
+		err    error
+		length int
+		hdr    textproto.MIMEHeader
+	)
+
+	resp := new(Event)
+	hdr, err = h.textreader.ReadMIMEHeader()
 	if err != nil {
-		h.err <- err
+		h.errEv <- err
 		return false
 	}
```

Comment:
This error might occur for `command/reply` or `api/response` where we are waiting for a response on `errReq` chan but sending data on `errEv`.

Reply:
True. but worst case it times out & still returns an error.